cypress/e2e/SiteWideSearchAppIntegration.feature

```diff
@@ -13,7 +13,6 @@ Feature: Site Wide Search app is integrated within CGOV platform
         And the "More information on dictionary page" button is displayed within the dictionary definition
         Then the "Show full definition" button within the dictionary definition is displayed
         And the "H4" title is "Results XXXXX-XXXXX of XXXXX for: mantle cell lymphoma"
-        And both pagers are displayed
```

Comment:
I see "And both pagers are displayed" is removed in multiple places which makes sense for removing the test. Should there be code removed as well?  Or are there checks for multiple pagers elsewhere?

Reply:
You are right - I thought it was reusing the common step def, but it turns out pager code had been implemented separately in sws index.js - removed!